libraries/core-react/src/components/Input/Input.tsx

```diff
@@ -9,10 +9,11 @@ import {
 } from '../../utils'
 import type { Variants } from '../TextField/types'
 import { useEds } from '../EdsProvider'
+import { useToken } from '../../hooks'
 
 const { input, inputVariants } = tokens
 
-const Variation = ({ variant, token, density }: StyledProps) => {
+const Variation = ({ variant, token }: StyledProps) => {
```

Comment:
`token` is the same as `theme` here, so suggest changing it to use theme instead of token.

Reply:
I tried to replace `token` with `theme` but it won't compile.. 🤔  I guess it could be because token=`inputVariant` and theme=`input`..?

Reply:
Merged variant tokens with input token, so it works now